src/app/views/query-response/snippets/snippets-helper.tsx

```diff
@@ -59,6 +70,42 @@ function Snippet(props: ISnippetProps) {
     dispatch(getSnippet(language));
   }, [sampleQuery.sampleUrl]);
 
+  const setCommentSymbol = (): string => {
+    return language.trim() === 'powershell' ? '#' : '//';
+  }
+
+  const trackLinkClickedEvent = (link: string, e:any) => {
+    const isDocumentationLink : boolean = link.includes('doc')
+    const componentName = isDocumentationLink ? componentNames.CODE_SNIPPET_DOCUMENTATION_LINK :
+      componentNames.CODE_SNIPPET_SDK_LIBRARY_LINK
+    telemetry.trackLinkClickEvent(e.currentTarget.href, componentName);
+  }
+
+  const extraSnippetInformation = () : JSX.Element => {
+    return (
+      <div style={{marginLeft: '28px', color: '#008000',
```

Comment:
There's lots of color repetition. Since we have the theme colours stored, add the 'green' color in our `dark.ts` and `light.ts` files.
Also the color has a contrast issue in dark mode which affects accessibility.
![image](https://user-images.githubusercontent.com/18407044/160615435-4a5a92a7-43b2-44f9-9851-64dd9882085b.png)

Perhaps you can save a different value in the `dark.ts` file such as `green: '#00a100'`
![image](https://user-images.githubusercontent.com/18407044/160615741-8fca677c-ce04-4847-a519-38f454150099.png)

Reply:
Nice catch. Let me fix that